convert2rhel/unit_tests/actions/system_checks/tainted_kmods_test.py

```diff
@@ -121,37 +121,35 @@ def test_check_tainted_kmods_skip(monkeypatch, command_return, is_error, tainted
     tainted_kmods_action.run()
 
     if is_error:
-        expected = set(
-            (
-                actions.ActionMessage(
-                    level="WARNING",
-                    id="TAINTED_KMODS_DETECTED_MESSAGE",
-                    title="Tainted kernel modules detected",
-                    description="Please refer to the diagnosis for further information",
-                    diagnosis=(
-                        "Tainted kernel modules detected:\n  system76_io\n  system76_acpi\nThird-party "
-                        "components are not supported per our software support"
-                        " policy:\n{}\n".format(LINK_KMODS_RH_POLICY)
-                    ),
-                    remediations=(
-                        "Prevent the modules from loading by following {0}"
-                        " and run convert2rhel again to continue with the conversion."
-                        " For information on what a tainted kernel module is, please refer to this documentation {1}".format(
-                            LINK_PREVENT_KMODS_FROM_LOADING, LINK_TAINTED_KMOD_DOCS
-                        )
-                    ),
+        expected = {
```

Comment:
I just wonder if we are changing the `set()` (which was in some examples for porting to actions, that's why it's used a lot) to `{}`, we probably should change it everywhere.

Or you just changed it in places where you saw that as a start of this change? In that case, changing it everywhere won't be needed it's kickoff :)

Reply:
> Or you just changed it in places where you saw that as a start of this change? In that case, changing it everywhere won't be needed it's kickoff :)

It was like that indeed.